charts/unspec-service/values.preview.template.yaml

```diff
@@ -1,12 +1,45 @@
 tags:
   unspec-ccd-stack: true
 
+postgresql:
+  enabled: true
+  image:
+    tag: '11.3.0'
+  resources:
+    requests:
+      cpu: 25m
+  initdbScripts:
+    init.sql: |-
+      CREATE DATABASE cmc WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "data-store" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "definition-store" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "user-profile" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+  postgresqlUsername: hmcts
+  postgresqlPassword: hmcts
+  persistence:
+    enabled: false
+
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  keyVaults:
+    unspec:
+      resourceGroup: unspec
+      secrets:
+        - unspec-idam-client-secret
+        - microservicekey-unspec-service
+        - system-update-user-username
+        - system-update-user-password
+        - AppInsightsInstrumentationKey
+        - docmosis-api-key
   environment:
     CORE_CASE_DATA_API_URL: http://${SERVICE_NAME}-ccd-data-store-api
     OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-aat2.internal:8443/openam/oauth2/realms/root/realms/hmcts
+    CMC_DB_HOST: ${SERVICE_NAME}-postgresql
+    CMC_DB_USERNAME: hmcts
+    CMC_DB_PASSWORD: hmcts
+    CMC_DB_CONNECTION_OPTIONS: '?sslmode=allow'
+    REFERENCE_DATABASE_MIGRATION: true
```

Comment:
all of this needed? stuff in values.yaml

Reply:
MArking this resolved as already discussed between @dharmendrak  and @swalker125